src/h5web/providers/mock/utils.test.ts

```diff
@@ -1,14 +1,11 @@
-import type { Group } from '../models';
 import { mockMetadata } from './metadata';
 import { findMockEntity } from './utils';
 
 describe('findMockEntity', () => {
   it('should return entity at given path', () => {
     expect(findMockEntity('/')).toBe(mockMetadata);
-    expect(findMockEntity('/nD_datasets')).toBe(mockMetadata.children[1]);
-    expect(findMockEntity('/entities/raw')).toBe(
-      (mockMetadata.children[0] as Group).children[4]
-    );
+    expect(findMockEntity('/nD_datasets').kind).toBe('group');
+    expect(findMockEntity('/entities/raw').name).toBe('raw');
```

Comment:
I keep having to fix this test when I make changes to the mock entities, so it's time I made it more reliable...